scripts/build/clear-builder-volumes

```diff
@@ -2,6 +2,8 @@
 
 set -euo pipefail
 
+# clear all the volumes that we use
```

Comment:
The formatting I've been using for these is:

- Capitalize 1-line description
- newline
- other notes

So in this case it would be

```
set -euo pipefail

Clear all the volumes that we use

Must be run from outside (stopped) container

docker volume ...
```

We should probably find somewhere to put coding guidelines so that we stay consistent when the automated formatters don't help.